dataline-config/models/src/main/resources/json/StandardTapConfig.json

```diff
@@ -19,10 +18,6 @@
     },
     "state": {
       "$ref": "State.json"
-    },
-    "standardSyncSummary": {
```

Comment:
This isn't used anywhere, is there a reason to keep it?

Reply:
nope. you can trash it.